tests/link/jax/test_tensor_basic.py

```diff
@@ -191,3 +191,27 @@ def test_jax_eye():
     out_fg = FunctionGraph([], [out])
 
     compare_jax_and_py(out_fg, [])
+
+
+def test_tri():
+    out = at.tri(10, 10, 0)
+    fgraph = FunctionGraph([], [out])
+    compare_jax_and_py(fgraph, [])
```

Comment:
Maybe worth testing with a symbolic `k`?

Reply:
Hmm it seems like it does :D

Reply:
Was it complaining because of the check, or because of JAX itself?